example/lib/ui/lang_page.dart

```diff
@@ -87,7 +87,7 @@ class _LanguagePageState extends State<LanguagePage> with SingleTickerProviderSt
   // * An example of a vertically reorderable list.
   Widget _buildVerticalLanguageList() {
     final theme = Theme.of(context);
-    const listPadding = EdgeInsets.symmetric(horizontal: 0);
+    const listPadding = EdgeInsets.only(right: 30);
```

Comment:
```suggestion
const listPadding = EdgeInsets.zero;
```
Is there a reason, you only applied padding to the right of the list?

Reply:
I was tested with iPhone 11 real device and I cannot long press right edge as following.
https://github.com/flutter/flutter/issues/67252
I don't know whether it is general problem or only my problem..

This is just for testing and doesn't matter

Reply:
Ok, thanks.